### PR TITLE
New getting started for windows/mac/linux

### PR DESCRIPTION
**Pull request by cartermp**
* Remove .NET Framework and Mono from the equation, all they do is complicate things
* Standardize on CLI tools first, then Ionide, then commercial IDEs on each OS

### CODE REVIEW

File: guides/web/index.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ and mobile apps with [React native](https://facebook.github.io/react-native/).
 #### Giraffe
 {: #giraffe  .anchor }
 
-[Giraffe](https://github.com/dustinmoris/Giraffe) is an F# micro web framework for building rich web applications. It has been heavily inspired and is similar to [Suave](https://suave.io/), but has been specifically designed with [ASP.NET Core](https://www.asp.net/core) in mind and can be plugged into the ASP.NET Core pipeline via [middleware](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/middleware).
+[Giraffe](https://github.com/dustinmoris/Giraffe) is an F# library for building rich web applications that use ASP.NET Core under the hood. It has been heavily inspired and is similar to [Suave](https://suave.io/), but has been specifically designed with [ASP.NET Core](https://www.asp.net/core) in mind and can be plugged into the ASP.NET Core pipeline via [middleware](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/middleware).
```
Comment (slang25) [inline guides/web/index.md:72]: https://github.com/giraffe-fsharp/Giraffe

File: use/linux/index.md [hunk 2]
```diff
@@ -1,208 +1,64 @@
 ---
 layout: default
-title: Use F# on Linux | The F# Software Foundation
-headline: Use F# on Linux
+title: Use F# on macOS | The F# Software Foundation
+headline: Use F# on macOS
```
Comment (slang25) [inline use/linux/index.md:4]: Edited in error?

File: use/linux/index.md [hunk 3]
```diff
@@ -1,208 +1,64 @@
...
 ---
 
+### Option 1: Install .Net Core SDK and use your own editor
```
Comment (slang25) [inline use/linux/index.md:7]:
```suggestion
### Option 1: Install .NET Core SDK and use your own editor
```

File: use/linux/index.md [hunk 4]
```diff
@@ -1,208 +1,64 @@
...
 
-## Option 1: Install from your Linux distribution's package manager
+Install [.NET Core SDK](https://dotnet.microsoft.com/download). .NET Core is available for major Linux distributions and is typically installed with the system package manager of your distribution of choice. Once that is installed, you can begin using F#!
 
-When being used as .NET language, F# requires a .NET runtime. On Linux this runtime is [Mono](http://www.mono-project.com/) or [.NET Core](https://docs.microsoft.com/en-us/dotnet/).
-Most Linux distributions include a version of Mono in their repositories. However, these versions
-are typically pretty old. Therefore, it is *highly recommended* that you add the official Mono package
-repository to your package manager, as described below. This makes it easy to get updates or uninstall, if 
-required.
+#### Say hello from a F# Script
```
Comment (slang25) [inline use/linux/index.md:11]:
```suggestion
#### Say hello from an F# Script
```

File: use/linux/index.md [hunk 5]
```diff
@@ -1,208 +1,64 @@
...
+If this is your first time using .NET Core, there will be a short, one-time message about using the .NET SDK. After that, you'll see the following output in your console:
 
-2. Install the `fsharp` package, which will pull in an updated version of Mono, if required.
+```
+Hello World from F#
+```
 
-        sudo apt-get update
-        sudo apt-get install fsharp
+#### Say hello from a F# Console app
```
Comment (slang25) [inline use/linux/index.md:31]:
```suggestion
#### Say hello from an F# Console app
```

File: use/mac/index.md [hunk 6]
```diff
@@ -1,109 +1,70 @@
...
-
-### Option 3: Install JetBrains Rider
-
-![logo](/images/thumbs/rider.png)&nbsp;[JetBrains Rider](https://www.jetbrains.com/rider) is a cross-platform .NET IDE built using IntelliJ and ReSharper technology. It offers support for .NET and .NET Core applications on all platforms.
+Install [.NET Core SDK](https://dotnet.microsoft.com/download). Once that is installed, you can begin using F#!
 
-1. Install [JetBrains Rider](https://www.jetbrains.com/rider/download/) for Mac.
-2. Install the [latest version of Mono](https://www.mono-project.com/download/stable/#download-mac).
-3. Ensure Mono `/bin` folder is added to your `PATH`: `export PATH=$PATH:/Library/Frameworks/Mono.framework/Versions/Current/bin/`. For more info see: [Mono installed on Mac, but not recognized in terminal](http://stackoverflow.com/questions/32542535/mono-installed-on-mac-but-not-recognized-in-terminal)
+#### Say hello from a F# Script
```
Comment (slang25) [inline use/mac/index.md:11]:
```suggestion
#### Say hello from an F# Script
```

File: use/mac/index.md [hunk 7]
```diff
@@ -1,109 +1,70 @@
...
+If this is your first time using .NET Core, there will be a short, one-time message about using the .NET SDK. After that, you'll see the following output in your console:
 
-To use the F# command-line compiler via Mono and tools on Mac OSX:
+```
+Hello World from F#
+```
 
-*  [Install Mono](https://www.mono-project.com/download/stable/#download-mac). Use version 4.2.0 or later.
-*  Add Mono `/bin` folder to your `PATH`: `export PATH=$PATH:/Library/Frameworks/Mono.framework/Versions/Current/bin/`. For more info see: [Mono installed on Mac, but not recognized in terminal](http://stackoverflow.com/questions/32542535/mono-installed-on-mac-but-not-recognized-in-terminal)
+#### Say hello from a F# Console app
```
Comment (slang25) [inline use/mac/index.md:31]:
```suggestion
#### Say hello from an F# Console app
```

File: use/windows/index.md [hunk 8]
```diff
@@ -4,59 +4,69 @@ title: Use F# on Windows | The F# Software Foundation
...
 ---
 
-### Option 1: Install .Net Core SDK and use your own editor.
+### Option 1: Install .Net Core SDK and use your own editor
 
-Install [.NET Core SDK](https://dotnet.microsoft.com/download). Once that is installed lets start by saying hello world.
-<br/>
+Install [.NET Core SDK](https://dotnet.microsoft.com/download). Once that is installed, you can begin using F#!
 
 #### Say hello from a F# Script
```
Comment (slang25) [inline use/windows/index.md:11]:
```suggestion
#### Say hello from an F# Script
```

File: use/windows/index.md [hunk 9]
```diff
@@ -4,59 +4,69 @@ title: Use F# on Windows | The F# Software Foundation
...
+```
 
-* [Visual Studio 2019](https://www.visualstudio.com/downloads/) comes with F# support in all its editions: Community, Professional and Enterprise. Community is completely free. See [Get started with F# in Visual Studio](https://docs.microsoft.com/dotnet/fsharp/get-started/get-started-visual-studio) for more information.
+If this is your first time using .NET Core, there will be a short, one-time message about using the .NET SDK. After that, you'll see the following output in your console:
+
+```
+Hello World from F#
+```
+
+#### Say hello from a F# Console app
```
Comment (slang25) [inline use/windows/index.md:31]:
```suggestion
#### Say hello from an F# Console app
```

File: guides/web/index.md [hunk 10]
```diff
@@ -69,7 +69,7 @@ and mobile apps with [React native](https://facebook.github.io/react-native/).
...
 
 * [Docs](https://github.com/dustinmoris/Giraffe#basics)
```
Comment (slang25) [inline guides/web/index.md:74]:
```suggestion
* [Docs](https://github.com/giraffe-fsharp/Giraffe/blob/master/DOCUMENTATION.md)
```

File: guides/web/index.md [hunk 11]
```diff
@@ -69,7 +69,7 @@ and mobile apps with [React native](https://facebook.github.io/react-native/).
...
 * [Samples](https://github.com/dustinmoris/Giraffe/tree/master/samples)
```
Comment (slang25) [inline guides/web/index.md:75]:
```suggestion
* [Samples](https://github.com/giraffe-fsharp/Giraffe/tree/master/samples)
```

File: use/linux/index.md (hunk 2 above)
Comment (cartermp) [reply use/linux/index.md:4]: oops, will fix

**Comment by dsyme** [comment]
I renamed `use/html5/index.md` to `use/webapps/index.md`.  Could you integrate that?

File: use/windows/index.md [hunk 12]
```diff
@@ -4,59 +4,69 @@ title: Use F# on Windows | The F# Software Foundation
...
 ![logo](/images/thumbs/rider.png)&nbsp;[JetBrains Rider](https://www.jetbrains.com/rider) is a cross-platform .NET IDE built using IntelliJ and ReSharper technology. It offers support for .NET and .NET Core applications on all platforms.
 
 1. Install [JetBrains Rider](https://www.jetbrains.com/rider/download/) for Windows.
-2. (optional) Install the latest [.NET Core SDK](https://www.microsoft.com/net/core#windowscmd).
-
-You will also need either to install Visual Studio or to follow the steps [here](https://rider-support.jetbrains.com/hc/en-us/articles/207288089-Using-Rider-under-Windows-without-Visual-Studio-prerequisites).
+2. Install the latest [.NET Core SDK](https://www.microsoft.com/net/core#windowscmd).
 
-<br />
+You will also need either to install Visual Studio or to follow the steps [here](https://rider-support.jetbrains.com/hc/en-us/articles/207288089-Using-Rider-under-Windows-without-Visual-Studio-prerequisites) if you need to use the older .NET Framework instead of .NET Core.
```
Comment (dsyme) [inline use/windows/index.md:68]: Still a mention of .NET Framework here.

Comment (cartermp) [reply use/windows/index.md:68]: curses!

Comment (cartermp) [reply use/windows/index.md:68]: Though I think this one is okay to include since a lot of folks still need to use .NET Framework

**Comment by dsyme** [comment]
I'll merge and check on the site

**Pull request merged**